### DISCUSSION TITLE
After the installation is complete, it prompts Corn Disabled and task scheduling disabled.

### DISCUSSION THREAD

**Discussion by Tony1129** [Blair2004/NexoPOS#1752] [Q&A]
After the message popped up, I clicked on it, and it led me to the Queues Jobs Are Disabled interface on the official website, but its explanation was too vague. I am a newbie and could not find a suitable tutorial on the Internet to guide me to fix it, so please Can you guide me in detail to fix these bugs?
![2024-02-01 103500](https://github.com/Blair2004/NexoPOS/assets/106466305/1e2b77bd-bd12-4def-90d7-7772497808e6)
![2024-02-01](https://github.com/Blair2004/NexoPOS/assets/106466305/a89cdbb9-bd76-47c6-9ae8-6f674fd9cc43)

**Comment by Blair2004** [comment]
The steps are the same as configuring Task Scheduling on Laravel https://laravel.com/docs/10.x/scheduling
Check this out.
